ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuiteRunner.scala

```diff
@@ -129,9 +126,15 @@ final class LedgerTestSuiteRunner(
       .flatMap(suite => suite.tests.map(suite -> _))
       .zipWithIndex
     Source(tests)
+      .throttle(3, 1.second)
```

Comment:
I understand why you had to do it, but I'd recommend:
1. making it clear with a comment
2. opening a ticket to track this if this can't be fixed in this PR